esiclient/tests/unit/v1/test_node_network.py

```diff
@@ -24,7 +24,7 @@ class TestList(base.TestCommand):
 
     def setUp(self):
         super(TestList, self).setUp()
-        self.cmd = node_network.List(self.app, None)
+        self.maxDiff = None
```

Comment:
I don't think this is needed (at least from my testing).

Reply:
Ah that was for my debugging, I'll get rid of it now